src/main/java/org/commcare/formplayer/beans/SessionNavigationBean.java

```diff
@@ -118,6 +120,9 @@ public void setIsPersistent(boolean persistent) {
     public boolean getIsShortDetail() {
         return isShortDetail;
     }
+    public boolean getIsRefreshCaseSearch() {
```

Comment:
I thought standard naming leaves the get off the name if the field is a boolean.

Reply:
yea, although the naming for response beans is a little different as  this gets used by the Json coverter to convert a json for this class and the standand naming convention there is `get_{$var}` and `set_{$var}`

Reply:
Does this need a setter or does it function differently from `isShortDetail`?

Reply:
it does, corrected